README.md

```diff
@@ -7,7 +7,7 @@ Sunrise Java :sunrise:
 The next generation shop template.
 
 * Demo: https://sunrise.commercetools.com
-* [Documentation](https://github.com/commercetools/commercetools-sunrise-java/blob/master/ONBOARDING.md)
+* [Documentation](https://github.com/commercetools/commercetools-sunrise-java/blob/master/manual/)
```

Comment:
Nice to know! I didn't give it much thought, I just modified it. I'll change it.